anitya/templates/search.html

```diff
@@ -88,6 +88,7 @@
     <blockquote>
         Oups this is embarrassing, it seems that no projects are being
         monitored currently.
+        <p><a href="/project/new?name={{ pattern }}">Click Here</a> to add this project instead. </p>
```

Comment:
You need to use the ``url_for()`` construct from flask as otherwise it would break if anitya isn't deployed at the root of your domain name